scripts/mk_usb_installer.sh

```diff
@@ -41,7 +45,9 @@ curl -OJ ${iso_loc}/SHA256SUMS
 grep ${iso} SHA256SUMS > ${iso}.SHA256SUM
 sha256sum --check ${iso}.SHA256SUM
 
-zcat hd-media/boot.img.gz|sudo dcfldd of=/dev/${dev}
+cd -
```

Comment:
I generally prefer using subshells to cd-ing around in a script, keeps it easier to follow the current directory...